packages/bpk-component-image/package.json

```diff
@@ -16,7 +16,8 @@
     "bpk-component-spinner": "^2.1.23",
     "bpk-mixins": "^16.5.5",
     "bpk-react-utils": "^2.3.4",
-    "prop-types": "^15.5.8"
+    "prop-types": "^15.5.8",
+    "react-addons-css-transition-group": "^15.6.0"
```

Comment:
We already have `react-addons-css-transition-group` listed as a dependency within `bpk-react-utils` - perhaps it's easier to control versioning and usage from one place? Thoughts?